### ISSUE TITLE
Search is not intuitive

### ISSUE THREAD

**Issue by notesboy** [teamstudio/xcontrols-domino#270]
When implementing the "Search ..." functionality for views using any of the View XPages (UnpBootFlatView for example) it only works if the entire word is entered.  

Example:
- Company Name = Roberts Cool Coding
- Search for "Roberts" returns results
- Search for "Rob" or Robert" does not return results.

Knowing that the results come from a categorized column in a view, is there any method for returning partial matches as detailed above?

**Comment by reidcanavan** [comment]
Or perhaps could the type-ahead feature be utilized to fill in the rest?

**Comment by notesboy** [comment]
We did consider this but did not want to modify the underlying Custom
Controls because we want all of the "Goodness" that will come in the future
of the xControls project.  If we make mods to the Custom Control then we
will need to re-apply them when updates arrive.


To alleviate the problem we have modified the first column of the search
view to have the following @Formula:

@Trim(@Unique(@Left(FirstName:LastName;2):@Left(FirstName:LastName;3):(FirstName:LastName))))

This way the user can search on the initial 2 or 3 characters or the whole
field value.  Not as elegant because now the view index is larger, but most
users will accept these parameters for searching.
